tests/e2e/filelog-receiver/00-install.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: filelog
+spec:
+  mode: sidecar
+  image: otel/opentelemetry-collector-k8s
+  volumeMounts:
+  - name: logs
```

Comment:
In this case the app is putting logs in to an emptydir 

```
      - name: logs
        emptyDir: {}
```

Can the collector scrape logs which are in a file that is not in a separate volume?

Reply:
Not that I am aware of.